packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -727,6 +760,33 @@ export class ApexDebug extends LoggingDebugSession {
       }
     } catch (error) {
       this.tryToParseSfError(response, error);
+      // telemetry for expired session or invalid org-isv-debugger-sid (authentication error)
```

Comment:
This block covers two cases that display the same popup and error message:
1. The `forceIde://` URL of the current ISV Debugger session has expired (pretty common use case, as the `forceIde://` URL expires after 5 hours of use and after the user has been idle for too long)
2. The `org-isv-debugger-sid` config variable is set to an invalid value (edge case, as the config variable should be automatically set to the correct value during the bootstrap process)